### PR TITLE
Es6

### PR DESCRIPTION
**Pull request by Enoisong**
### In this project, I:
- Recreate my Awesome Book App
- Divide the code into Modules
- Use Es6 syntax
- Refactor the functions to arrow functions
- Display current date below the navigation bar,

### CODE REVIEW

File: modules/list.js [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
     }
 
     removeBook = (id) => {
-        this.books = this.books.filter((book) => book.id !== Number(id));
-        this.saveBooks();
-        this.renderBooks();
-        let count = Number(localStorage.getItem('count'));
-        count -= 1;
-        localStorage.setItem('count', count);
+      this.books = this.books.filter((book) => book.id !== Number(id));
```
Comment (Davidosky007) [inline modules/list.js:13]: * Nice Job so far 👏 just one little bug I noticed when I try to delete the first book added which doesn't respond below is a recording of the bug 👇 

https://user-images.githubusercontent.com/52497246/229973860-1fa0ea0d-c1df-48bf-a8f1-73137beeede7.mp4

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Enoisong 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Proper use of ES6 syntax 👍 
* ES6 modules used properly 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Enoisong** [comment]
Hi Reviewer, I got  your recording of the changes to be made in my
submitted review https://github.com/Enoisong/AWESOMEBOOKS-ES6/pull/1. Thank
you so much for the observation. Please I want to remark that in my pc all
the inputted names are removed on click. Could you please offer any
suggestions on how to go about this, as I am reluctant to send in for a
second review? Thank you.

**Review by julie-ify — APPROVED** [review]
Hi @Enoisong,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**